webaverse/app
# Footstep sound effect synchronization

Related issue:
https://github.com/webaverse/app/issues/1909

Steps for testing:
Footsteps for walk,run,naruto run should be synced with animation

In fly mode there should not be any sound (jump, jump end etc)
